calico/felix/actor.py

```diff
@@ -338,6 +338,7 @@ def _finish_msg_batch(self, batch, results):
         pass
 
     def _maybe_yield(self):
+    # MD4: Premature optimization or really worth including?
```

Comment:
I think adding this is just being a good citizen.  E.g. if we're doing a big apply_snapshot, we should yield the thread from time to time to allow etcd polling and iptables commands to run.